root/etc/s6-overlay/s6-rc.d/init-wireguard-confs/run

```diff
@@ -56,8 +56,8 @@ DUDE"
         fi
       else
         for idx in {2..254}; do
-          PROPOSED_IP="${INTERFACE}.${idx}"
-          if ! grep -q -R "${PROPOSED_IP}" /config/peer*/*.conf 2>/dev/null && ([[ -z "${ORIG_INTERFACE}" ]] || ! grep -q -R "${ORIG_INTERFACE}.${idx}" /config/peer*/*.conf 2>/dev/null); then
+          PROPOSED_IP="${INTERFACE}.${idx}/32"
+          if ! grep -q -R "${PROPOSED_IP}" /config/peer*/*.conf 2>/dev/null && ([[ -z "${ORIG_INTERFACE}" ]] || ! grep -q -R "${ORIG_INTERFACE}.${idx}/32" /config/peer*/*.conf 2>/dev/null); then
```

Comment:
@thespad, is something like this going to be better? If not, do you have any suggestions?

```suggestion
for idx in {2..254}; do
  PROPOSED_IP="${INTERFACE}.${idx}/32"
  if ! grep -q -R "${INTERFACE}.${idx}" /config/peer*/*.conf 2>/dev/null &&
     ! grep -q -R "${INTERFACE}.${idx}/32" /config/peer*/*.conf 2>/dev/null &&
     ([[ -z "${ORIG_INTERFACE}" ]] || 
      (! grep -q -R "${ORIG_INTERFACE}.${idx}" /config/peer*/*.conf 2>/dev/null &&
       ! grep -q -R "${ORIG_INTERFACE}.${idx}/32" /config/peer*/*.conf 2>/dev/null)); then
    CLIENT_IP="${PROPOSED_IP}"
    break
  fi
done
```